src/ParallelAlgorithms/Events/ObserveVolumeIntegrals.hpp

```diff
@@ -120,8 +121,9 @@ class ObserveVolumeIntegrals<
 
   using argument_tags = tmpl::list<
       ::Tags::ObservationBox, ObservationValueTag,
-      domain::Tags::Mesh<VolumeDim>,
-      domain::Tags::DetInvJacobian<Frame::ElementLogical, Frame::Inertial>,
+      ::Events::Tags::ObserverMesh<VolumeDim>,
+      ::Events::Tags::ObserverDetInvJacobian<Frame::ElementLogical,
+                                           Frame::Inertial>,
```

Comment:
Could you run clang-format over this? What I usually do is run git-clang-format when I have staged changes, or something like `git-clang-format HEAD~1` to format the last commit.